packages/extension-ordered-list/src/ordered-list.ts

```diff
@@ -84,6 +84,10 @@ export const OrderedList = Node.create<OrderedListOptions>({
             : 1
         },
       },
+      type: {
+        default: null,
+        parseHTML: element => element.getAttribute('type')
```

Comment:
Sorry minor fix. Could you handle this @bastianjoel

Reply:
Done